packages/console/src/components/CopyToClipboard/index.tsx

```diff
@@ -12,14 +14,24 @@ type Props = {
   value: string;
   className?: string;
   variant?: 'text' | 'contained' | 'border' | 'icon';
+  isContentHidden?: boolean;
 };
 
 type CopyState = TFuncKey<'translation', 'admin_console.general'>;
 
-const CopyToClipboard = ({ value, className, variant = 'contained' }: Props) => {
+const CopyToClipboard = ({ value, className, isContentHidden, variant = 'contained' }: Props) => {
   const copyIconReference = useRef<HTMLDivElement>(null);
   const [copyState, setCopyState] = useState<CopyState>('copy');
   const { t } = useTranslation(undefined, { keyPrefix: 'admin_console.general' });
+  const [showHiddenContent, setShowHiddenContent] = useState(false);
+
+  const valueToShow = useMemo(() => {
+    if (!isContentHidden || showHiddenContent) {
+      return value;
+    }
+
+    return '*'.repeat(value.length);
```

Comment:
Will the length always be 32? Just asking

Reply:
64